packages/myst-to-tex/src/container.ts

```diff
@@ -37,11 +46,13 @@ export function determineCaptionKind(node: GenericNode): CaptionKind | null {
 
 function nodeToCommand(node: Image | Table | Code | Math) {
   const kind = determineCaptionKind(node);
+  const classes = getClasses((node as any).class);
+  const fullWidth = classes.includes('full-width') || classes.includes('w-full');
   switch (kind) {
     case CaptionKind.fig:
-      return (node as any).fullpage ? 'figure*' : 'figure';
```

Comment:
Ah, cool, so before this was just an arbitrary attribute that was not referred to anywhere else.

Now, it just takes advantage of the existing container `class` property.